base/applications/cleanmgr/util.c

```diff
@@ -0,0 +1,1179 @@
+    {
+        DPRINT("RegOpenKeyExW(): Failed to open a registry key!\n");
+    }
+    
+    HWND hwnd = (HWND)lpParam;
+    
+    HWND hProgressBar = NULL;
+    WCHAR TargetedDir[MAX_PATH] = { 0 };
+    SHQUERYRBINFO RecycleBinInfo;
+    ZeroMemory(&RecycleBinInfo, sizeof(RecycleBinInfo));
+    RecycleBinInfo.cbSize = sizeof(RecycleBinInfo);
+
+    hProgressBar = GetDlgItem(hwnd, IDC_PROGRESS_1);
+    
+    if(bv.SysDrive == TRUE)
+    {
+        SHGetFolderPathW(NULL, CSIDL_WINDOWS, NULL, SHGFP_TYPE_CURRENT, TargetedDir);
+        StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\Temp");
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.TempASize = DirSizeFunc(TargetedDir);
+            if (sz.TempASize == 1)
+            {
+                return FALSE;
+            }
+        }
+        SHGetFolderPathW(NULL, CSIDL_PROFILE, NULL, SHGFP_TYPE_CURRENT, TargetedDir);
+        StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\Local Settings\\Temp");
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.TempBSize = DirSizeFunc(TargetedDir);
+            if (sz.TempBSize == 1)
+            {
+                return FALSE;
+            }
+            SendMessageW(hProgressBar, PBM_SETPOS, 25, 0);
+            SetDlgItemTextW(hwnd, IDC_STATIC_INFO, L"Temporary Files");
+        }
+    }
+    
+    /* Hardcoding the path temporarily */
+    
+    StringCchPrintfW(TargetedDir, _countof(TargetedDir), L"%s\\RECYCLED", wcv.DriveLetter);
+    if(!PathIsDirectoryW(TargetedDir))
+    {
+        ZeroMemory(&TargetedDir, sizeof(TargetedDir));
+        StringCchPrintfW(TargetedDir, _countof(TargetedDir), L"%s\\RECYCLER", wcv.DriveLetter);
+    }
+
+    sz.RecycleBinSize = DirSizeFunc(TargetedDir);
+    if (sz.TempBSize == 1)
+        return FALSE;
+
+    SendMessageW(hProgressBar, PBM_SETPOS, 50, 0);
+    SetDlgItemTextW(hwnd, IDC_STATIC_INFO, L"Recycled Files");
+    
+    
+    /* Currently disable because SHQueryRecycleBinW isn't implemented in ReactOS */
+    
+    /*StringCbCopyW(TargetedDir, _countof(TargetedDir), wcv.DriveLetter);
+    StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\");
+
+    if(SHQueryRecycleBinW(TargetedDir, &RecycleBinInfo) == S_OK)
+    {
+        sz.RecycleBinSize = RecycleBinInfo.i64Size;
+        SendMessageW(hProgressBar, PBM_SETPOS, 50, 0);
+        SetDlgItemTextW(hwnd, IDC_STATIC_INFO, L"Recycled Files");
+    }*/
+
+
+    for(int i = 0; i < 10; i++)
+    {
+        StringCchPrintfW(TargetedDir, _countof(TargetedDir), L"%s\\FOUND.%.3i", wcv.DriveLetter, i);
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.ChkDskSize += DirSizeFunc(TargetedDir);
+            if (sz.ChkDskSize == 1)
+            {
+                return FALSE;
+            }
+            SendMessageW(hProgressBar, PBM_SETPOS, 75, 0);
+            SetDlgItemTextW(hwnd, IDC_STATIC_INFO, L"Old ChkDsk Files");
+        }
+        
+        else
+            break;
```

Comment:
Nit: it's generally more readable to add braces to both branches of an `if`, if one requires them. Though these days we even gravitate towards a style where all blocks are enclosed in braces.
In this particular instance, you could also simply invert the condition:
```
if (!PathIsDirectory())
{
    break;
}
// other code
```